README.md

```diff
@@ -51,41 +58,108 @@ pip install -r requirements/base.txt
 
 By default [`python tools/train.py`](https://gitlab-icv.inn.intel.com/algo_rnd_team/anomaly/-/blob/development/train.py)
 runs [PADIM](https://arxiv.org/abs/2011.08785) model [MVTec](https://www.mvtec.com/company/research/datasets/mvtec-ad) `leather` dataset.
-```
+
+```bash
 python tools/train.py    # Train PADIM on MVTec leather
 ```
 
 Training a model on a specific dataset and category requires further configuration. Each model has its own configuration
 file, [`config.yaml`](https://gitlab-icv.inn.intel.com/algo_rnd_team/anomaly/-/blob/development/stfpm/anomalib/models/stfpm/config.yaml)
 , which contains data, model and training configurable parameters. To train a specific model on a specific dataset and
 category, the config file is to be provided:
-```
+
+```bash
 python tools/train.py --model_config_path <path/to/model/config.yaml>
 ```
 
 Alternatively, a model name could also be provided as an argument, where the scripts automatically finds the corresponding config file.
-```
+
+```bash
 python tools/train.py --model stfpm
 ```
+
 where the currently available models are:
-* [DFKDE](anomalib/models/dfkde)
-* [DFM](anomalib/models/dfm)
-* [PADIM](anomalib/models/padim)
-* [PatchCore](anomalib/models/patchcore)
-* [STFPM](anomalib/models/stfpm)
+
+- [DFKDE](anomalib/models/dfkde)
+- [DFM](anomalib/models/dfm)
+- [PADIM](anomalib/models/padim)
+- [PatchCore](anomalib/models/patchcore)
+- [STFPM](anomalib/models/stfpm)
 
 ## Inference
 
 Anomalib contains several tools that can be used to perform inference with a trained model. The script in [`tools/inference`](tools/inference.py) contains an example of how the inference tools can be used to generate a prediction for an input image.
 
 The following command can be used to run inference from the command line:
-```
+
+```bash
 python tools/inference.py --model_config_path <path/to/model/config.yaml> --weight_path <path/to/weight/file> --image_path <path/to/image>
 ```
+
 If the specified weight path points to a PyTorch Lightning checkpoint file (`.ckpt`), inference will run in PyTorch. If the path points to an ONNX graph (`.onnx`) or OpenVINO IR (`.bin` or `.xml`), inference will run in OpenVINO.
 
```

Comment:
And here, it would be nice if we sum these up to a single complete example such as,

```bash
# Train a model on MVTec leather category.
python train.py tools/train.py --model padim

# Infer the model on MVTec leather color category.
python tools/inference.py \
    --model_config_path  anomalib/models/stfpm/config.yaml\
    --weight_path results/padim/mvtec/leather/weights/model.ckpt \
    --image_path datasets/MVTec/leather/test/color/000.png
```